### ISSUE TITLE
predicted mask format

### ISSUE THREAD

**Issue by Feanor007** [jocpae/clDice#21]
During training, should I convert the output from the Unet to a binary mask prior feeding into soft_dice_cldice? I used raw logits, but the training result is really bad.
However when I tried to convert to a binary mask, I lost the grad on the tensor.

**Comment by jocpae** [comment]
Hi, thanks for reaching out. To make it short, you should not convert the output to a binary mask. Did you train using only the dice loss as well (alpha=0? What do your results look like in this case? :)

**Comment by jocpae** [comment]
Good to hear :) 
I do not know what kind of data you are working on but in our experience, it was always a good idea to test a wide range of alpha values empirically to find out what best suits your dataset.
